spec/models/manageiq/providers/azure/cloud_manager_spec.rb

```diff
@@ -60,15 +60,25 @@
       @alt_client_id  = 'testuser'
       @alt_client_key = 'secret'
       @alt_tenant_id  = 'ABCDEFGHIJABCDEFGHIJ0123456789AB'
+
+      # Necessary since we're dealing with multiple cassettes
+      ::Azure::Armrest::ArmrestService.clear_caches
```

Comment:
Can we clear the caches in an after instead? It's usually more reliable to clean up after your own test rather than some others.  The next test we write that doesn't use this before could be contaminated if run after these tests.